libs/core/src/lib/card/card.component.ts

```diff
@@ -32,6 +32,9 @@ export class CardComponent implements OnChanges, OnInit, CssClassBuilder, OnDest
     /** Badge */
     @Input() badge: string;
 
+    /** Tabindex for badge */
+    @Input() badgeTabindex = '0';
+
```

Comment:
Three things:
1. Changes related to Card component should not be done in a PR addressing issues related to Carousel component. It's hard to track in future. Please move it to a separate PR.
2. When you move it to a separate PR I will request changes because Badge doesn't have to be focussed with the tab key and will ask the a11y team to comment on this
3. I am pretty sure there's already a PR for this.